packages/api/src/docker-compatibility-info.ts

```diff
@@ -42,3 +42,16 @@ export interface DockerSocketMappingStatusInfo {
     architecture: string;
   };
 }
+
+export interface DockerContextInfo {
```

Comment:
Could you add some comments on what this object is, and where the values came from? I must admit I feel a bit lost which the docker contexts as I've never used them

Reply:
I'll add a link to https://docs.docker.com/engine/manage-resources/contexts/

Reply:
Perfect thanks

Reply:
✅